test/MUXControlsTestApp/MUXControlsTestApp.Shared.projitems

```diff
@@ -146,6 +146,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="$(MSBuildThisFileDirectory)Assets\ladybug.wmv" />
```

Comment:
is this asset okay to use copywrite wise?

Reply:
This is the same we use in XCG.

Reply:
Yup, this is where I grabbed it.